OracleAnalytics/dockerfiles/buildDockerImage.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
```

Comment:
Please rename this file to `buildContainerImage.sh` and update all references accordingly.